### PR TITLE
tidy zen html

### PR DESCRIPTION
**Pull request by MeghanHughes**
Added indentation to the html in order to tidy up the file for future use.

### CODE REVIEW

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+                <a href="/202/" class="design-name">Retro Theater</a> by            <a href="#" class="designer-name">Eric Rogé</a>
+              </li>         <li>
+                <a href="/201/" class="design-name">Lily Pond</a> by            <a href="#" class="designer-name">Rose Thorogood</a>
+              </li>         <li>
+                <a href="/200/" class="design-name">Icicle Outback</a> by           <a href="http://www.timovirtanen.com/" class="designer-name">Timo Virtanen</a>
+              </li>         <li>
+                <a href="/199/" class="design-name">CSS ZEN ARMY</a> by           <a href="http://www.niceguy.com" class="designer-name">Carl Desmond</a>
+              </li>         <li>
+                <a href="/198/" class="design-name">The Original</a> by           <a href="#" class="designer-name">Joachim Shotter</a>
+              </li>         </ul>
```
Comment (mischa-saunders) [inline pristine-zen-garden.html]: Line 100 Alignment of </ul>

File: pristine-zen-garden.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            </nav>
+          </div>
+
```
Comment (mischa-saunders) [inline pristine-zen-garden.html]: The nav and divs could maybe be moved back to line up with the above tags they are closing

File: pristine-zen-garden.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                    <span class="indicator">&lsaquo;</span> Previous Designs              </a>
+                  </li>
+                  <li class="viewall">
+                    <a href="http://www.mezzoblue.com/zengarden/alldesigns/" title="View every submission to the Zen Garden.">
+                      View All Designs              </a>
+                    </li>
+                  </ul>
+                </nav>
+              </div>
+
```
Comment (mischa-saunders) [inline pristine-zen-garden.html]: Li Ul Nav and Div closing tags are all indented further than their opening tags

File: pristine-zen-garden.html [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+                          <li class="zen-translations">
+                            <a href="http://www.mezzoblue.com/zengarden/translations/" title="View translated versions of this page.">
+                              Translations            </a>
+                            </li>
+                          </ul>
+                        </div>
+                      </div>
+                    </aside>
+
+
```
Comment (mischa-saunders) [inline pristine-zen-garden.html]: Same thing down here... from line 147-157 the closing tags are deeper indents than the opening tags

File: pristine-zen-garden.html (hunk 1 above)
Comment (MeghanHughes) [inline pristine-zen-garden.html]: Good one!

File: pristine-zen-garden.html (hunk 4 above)
Comment (MeghanHughes) [inline pristine-zen-garden.html]: Thanks, Mischa those were great suggestions thanks for picking them up!

**Comment by mischa-saunders** [comment]
Hi! Do you want me to approve the pull request as is or do you want to look at lines 122,123,124,129-148, 150,151,152, to get the opening and closing tags aligned? Hope all is well!

**Comment by mischa-saunders** [comment]
There are still some small details that aren't quite clear to me but it is much better. One thing that helped me to really understand this is that I have started using the atom package "beautify" which arranges everything perfectly.

**Comment by mischa-saunders** [comment]
uh-oh I do not have write access so now I can't merge the pull request :(

**Comment by mischa-saunders** [comment]
ah figured it out

**Pull request merged**